File: Assignment 9/Q1.c
/* Write a Program which Accept Number from user and return the count of even digits.
   ex. Input: 2395
       Output: 1
       
       Input: -1018
       Output: 2
*/

#include<stdio.h>

int CountEven(int iNo)
{
    int iDigit = 0;
    int iCnt = 0;

    if(iNo < 0)   // Updator
    {
        iNo = -iNo;
    }

    while(iNo != 0)
    {
        iDigit = iNo % 10; 

        if(iDigit % 2 == 0)  // Check if even
        {
            iCnt++;
        }

        iNo = iNo / 10;      // Remove last digit
    }
    return iCnt; 
}

int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter Number : ");
    scanf("%d",&iValue);

    iRet = CountEven(iValue);

    printf("%d",iRet);

    return 0;
}

File: Assignment 17/Q1.java
// Write a program to find the Sum of Digits of a Number.

class Logic
{
    void sumOfDigits(int num)
    {
        int digit = 0;
        int sum = 0;

        while(num != 0)
        {
            digit = num % 10;
            sum = sum + digit;
            num = num / 10;    
        }

        System.out.println("Sum of digits = " + sum);
    }
}

class Q1
{
    public static void main(String args[])
    {
        Logic lobj = new Logic();
        lobj.sumOfDigits(1234);
    }
}